### PR TITLE
PHPUnit update - use better assertion names, remove deprecations

### PR DESCRIPTION
**Pull request by michalbundyra**

### CODE REVIEW

File: test/PackageVersionsTest/E2EInstallerTest.php [hunk 1]
```diff
@@ -179,7 +179,7 @@ public function testRemovingPluginWithNoDevDoesNotAttemptToGenerateVersions() :
 
         $this->execComposerInDir('install --no-dev', $this->tempLocalComposerHome);
 
-        $this->assertFileNotExists(
+        $this->assertFileDoesNotExist(
```
Comment (Ocramius) [inline test/PackageVersionsTest/E2EInstallerTest.php:182]: I think these are `self::`, right?

Comment (michalbundyra) [reply test/PackageVersionsTest/E2EInstallerTest.php]: Right.

**Comment by michalbundyra** [comment]
Why 1.9.0 not 1.8.X?

This is update for only dev (testing) deps and tests.

**Comment by Ocramius** [comment]
All improvements go to new minor releases: no bugs being fixed here ;-)

**Pull request merged**